src/platform/iotjs_module_uart-linux-general.inl.h

```diff
@@ -0,0 +1,168 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef IOTJS_MODULE_UART_LINUX_GENERAL_INL_H
+#define IOTJS_MODULE_UART_LINUX_GENERAL_INL_H
+
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/signal.h>
+#include <termios.h>
+#include <unistd.h>
+
+#include "module/iotjs_module_uart.h"
+
+void signal_handler_IO(int status);
+
+iotjs_jval_t jthis;
+int fd;
+uv_poll_t poll_handle;
+bool WRITING = false;
```

Comment:
1) I think we can put `static` not to pollute global name space.
2) Why do you use all capital variable name for `WRITING`? If there is a compelling reason, please change the name.